src/client/feature_proxy.rs

```diff
@@ -49,6 +49,10 @@ impl<'a> Feature for FeatureProxy<'a> {
     fn get_value(&self, entity: &impl Entity) -> crate::errors::Result<Value> {
         self.client.get_feature(&self.feature_id)?.get_value(entity)
     }
+
+    fn get_value_t<T: TryFrom<Value, Error = crate::Error>>(&self, entity: &impl Entity) -> crate::errors::Result<T> {
```

Comment:
`TryFrom<>`!! brilliant :) 
I was very unhappy with the boilerplate that was driving me the `Form<> + Form<> ...` 

I can imagine where the `_t` convention is coming from, looked very familiar to my C++ mindset :)
But I would definitely call it:
```
get_value_by_type()
```
and make it even more explicit. Does it make sense?